sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/PhoneNumbersImpl.java

```diff
@@ -465,42 +354,79 @@ public PagedFlux<PhoneNumberAreaCode> listAreaCodesAsync(
-            Integer skip,
-            Integer maxPageSize,
-            PhoneNumberAssignmentType assignmentType,
-            String locality,
-            String administrativeDivision,
-            String acceptLanguage,
-            Context context) {
+    public PagedFlux<PhoneNumberAreaCode> listAreaCodesAsync(String countryCode, PhoneNumberType phoneNumberType,
+        Integer skip, Integer maxPageSize, PhoneNumberAssignmentType assignmentType, String locality,
+        String administrativeDivision, String acceptLanguage, Context context) {
         return new PagedFlux<>(
-                () ->
-                        listAreaCodesSinglePageAsync(
-                                countryCode,
-                                phoneNumberType,
-                                skip,
-                                maxPageSize,
-                                assignmentType,
-                                locality,
-                                administrativeDivision,
-                                acceptLanguage,
-                                context),
-                nextLink -> listAreaCodesNextSinglePageAsync(nextLink, acceptLanguage, context));
+            () -> listAreaCodesSinglePageAsync(countryCode, phoneNumberType, skip, maxPageSize, assignmentType,
+                locality, administrativeDivision, acceptLanguage, context),
+            nextLink -> listAreaCodesNextSinglePageAsync(nextLink, acceptLanguage, context));
     }
 
     /**
      * Gets the list of available area codes.
-     *
+     * 
      * @param countryCode The ISO 3166-2 country code, e.g. US.
      * @param phoneNumberType Filter by numberType, e.g. Geographic, TollFree.
      * @param skip An optional parameter for how many entries to skip, for pagination purposes. The default value is 0.
      * @param maxPageSize An optional parameter for how many entries to return, for pagination purposes. The default
-     *     value is 100.
+     * value is 100.
      * @param assignmentType Filter by assignmentType, e.g. Person, Application.
      * @param locality The name of locality or town in which to search for the area code. This is required if the number
-     *     type is Geographic.
+     * type is Geographic.
+     * @param administrativeDivision The name of the state or province in which to search for the area code.
+     * @param acceptLanguage The locale to display in the localized fields in the response. e.g. 'en-US'.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws CommunicationErrorResponseException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the list of available area codes along with {@link PagedResponse}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PagedResponse<PhoneNumberAreaCode> listAreaCodesSinglePage(String countryCode,
```

Comment:
These are new APIs. Is this expected?

Reply:
Yes, this is expected. These APIs are in implementation package. So, this doesn't impact the public API surface.